src/components/Learn-Components/TOC-Learning-Path/index.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+// import { HiOutlineChevronLeft } from "react-icons/hi";
+// import { Link } from "gatsby";
+import TOCWrapper from "./toc.style";
+
+const TOC = ({ coursesData }) => {
+
+  // const reformatTOC= (data) => {
+  //   let newData = data.split("-").join(" ");
+  //   let firstLetter = newData.charAt(0).toUpperCase();
+  //   newData = `${firstLetter}${newData.slice(1)}`;
+  //   return newData;
+  // };
+
+  return (
+    <TOCWrapper>
+      {/** <div className="go-back">
+        <Link to={`/learn-ng`}>
+          <HiOutlineChevronLeft />
+          <h4>Learn-Layer5</h4>
+        </Link>
+      </div>*/}
+      <div className="toc-list">
+        <ul>
+          <h5 className="toc-sub-heading">
+            Pre-requisites
+          </h5>
+          {[{name: "Set up", link: "/service-mesh-management/meshery"},{name:"Run Meshery",link:"/service-mesh-management/meshery/getting-started"},{name: "Manage Meshery", link: "/service-mesh-management/meshery/operating-service-meshes"}].map(item =>
```

Comment:
These links should be a href to the sections within the page, and not point to the actual page.
Also, better to have these in a variable above and then use that variable here.